### ISSUE TITLE
[BUG]: kiwiiPost(s)

### ISSUE THREAD

**Issue by nfAdrian** [College-Kings/College-Kings#276]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Scene number

V7

### Bug Description

Just after that 'numberlikes' problem from #275, I went straight into another markdowns, related to v7_kiwiiPost1, 2, 3 and so on...

### Expected behaviour

The posts appearing on the Kiwii app, but the errors not showing at all:

![empty](https://user-images.githubusercontent.com/108369429/210822191-7c53bb75-e0ef-4641-9a69-baa4339bd1ab.png)

### Environment

```Markdown
- OS: Windows 11 Home
- Version: 22H2
```

### Additional context

```
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/v7/v7.rpy", line 1344, in script
    $ v7_kiwiiPost1.newComment(grayson, _("Check your DMs"), 14)
  File "game/v7/v7.rpy", line 1344, in <module>
    $ v7_kiwiiPost1.newComment(grayson, _("Check your DMs"), 14)
NameError: name 'v7_kiwiiPost1' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/v7/v7.rpy", line 1344, in script
    $ v7_kiwiiPost1.newComment(grayson, _("Check your DMs"), 14)
  File "D:\renpy\renpy\ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\renpy\renpy\python.py", line 1061, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/v7/v7.rpy", line 1344, in <module>
    $ v7_kiwiiPost1.newComment(grayson, _("Check your DMs"), 14)
NameError: name 'v7_kiwiiPost1' is not defined

Windows-10-10.0.22621 AMD64
Ren'Py 8.0.3.22090809
College Kings 1.2.8s (SHA: 08ace99dc)
Thu Jan  5 17:39:52 2023
```

### Which version of the app are you using?

1.2.0

**Comment by nfAdrian** [comment]
Forgot to mention: this error triggers for ALL the kiwii posts, apparently.

**Comment by nfAdrian** [comment]
Well... Looks like those errors were addicted to the (now fixed) #275 issue. However, the permanent notification on Kiwii has spread to CK1 too.
